test/Create3Deployer.js

```diff
@@ -154,6 +154,25 @@ describe('Create3Deployer', () => {
       expect(await contract.decimals()).to.equal(decimals);
     });
 
+    it('should deploy with passing native value to the contact', async () => {
```

Comment:
Should we add test for deployAndInit with native value?

Reply:
Yeah that's a good idea